src/main/kotlin/car/Application.kt

```diff
@@ -1,16 +1,27 @@
 package car
 
-import car.business.Cars
-import car.io.InputView
-import car.io.ResultView
+import car.domain.CarNames
+import car.domain.Cars
+import car.domain.Game
+import car.view.InputView
+import car.view.ResultView
 
 fun main() {
     val inputView = InputView()
-    val input = inputView.input()
 
-    val cars = Cars(input.splitNameBySplitter())
-    val resultView = ResultView(cars)
+    val names = inputView.inputNames()
+    val tryCount = inputView.inputTryCount()
+    val prettyString = inputView.inputPrettyString()
 
-    cars.move(input.tryCount)
-    resultView.print()
+    val carNames = CarNames(names.splitNameBySplitter())
+    val cars = Cars(carNames)
+
+    val game: Game = Game(cars)
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-racingcar/pull/158#discussion_r584423599

```suggestion
    val game = Game(Cars(CarNames(names.splitNameBySplitter())))
```

Reply:
아, 이 부분은 다시한번 새겨야겠어요~